packages/ubuntu_provision/lib/src/services/page_config_service.freezed.dart

```diff
@@ -138,7 +138,7 @@ class _$PageConfigEntryImpl implements _PageConfigEntry {
   }
 
   @override
-  bool operator ==(dynamic other) {
+  bool operator ==(Object other) {
```

Comment:
Are those unintended changes caused by the linter? Freezed seems to generate those lines with a `dynamic` type.

Reply:
The new version generates `Object` instead of `dynamic`, it's strange that it isn't `Object?` though...

Reply:
Hm, I was certain I ran `pub upgrade`.. I'll check again :)